data-types/strings/tests/test_strings.py

```diff
@@ -7,71 +7,72 @@ def test_one(self):
 
         # In place of the line below, use a standard function to capitalize name
         # actual = <your code here>
-        actual = None
+        actual = name.capitalize()
+
 
         expected = "Alice"
 
         assert actual == expected
 
-    @pytest.mark.skip
+
     def test_two(self):
         name = "aLiCe"
         # In place of the line below, call a method to achieve the expected output.
-        actual = None
+        actual = name.upper()
         expected = "ALICE"
 
         assert actual == expected
 
-    @pytest.mark.skip
+
     def test_three(self):
         name = "AlIcE"
         # In place of the line below, call a method to achieve the expected output.
-        actual = None
+        actual = name.lower()
         expected = "alice"
 
         assert actual == expected
 
-    @pytest.mark.skip
+
     def test_four(self):
         rhyme = "peter piper picked a peck of picked peppers"
         # In place of the line below, call a method to achieve the expected output.
-        actual = None
+        actual = rhyme[::-1]
         expected = "sreppep dekcip fo kcep a dekcip repip retep"
 
         assert actual == expected
 
-    @pytest.mark.skip
+
     def test_five(self):
         word = "ticking"
         # In place of the line below, call a method to achieve the expected output.
-        actual = None
+        actual = word.replace("t", "k")
         expected = "kicking"
 
         assert actual == expected
 
-    @pytest.mark.skip
+
     def test_six(self):
         word = "ticking"
         # In place of the line below, call a method to achieve the expected output.
-        actual = None
+        actual = word.replace("ticking", "clocking")
         expected = "clocking"
 
         assert actual == expected
 
-    @pytest.mark.skip
+
     def test_seven(self):
         words = "five sleepy kittens"
         # In place of the line below, call a method to achieve the expected output.
-        actual = None
+        actual = words.replace("e", "*")
         expected = "fiv* sl**py kitt*ns"
 
         assert actual == expected
 
-    @pytest.mark.skip
+
     def test_eight(self):
         greeting = "Hello!!"
         # In place of the line below, call a method to achieve the expected output.
-        actual = None
+        actual = slice(-1)
```

Comment:
I think you are on the right track here. Remember, strings are lists of characters so you can use the same list operations on them as you would on a normal list [(list slicing)](https://stackoverflow.com/questions/509211/understanding-slicing)